src/fde.h

```diff
@@ -184,6 +210,9 @@ class fde
     // I/O methods connect Squid to the device/stack/library fde represents
     READ_HANDLER *readMethod_ = nullptr; ///< imports bytes into Squid
     WRITE_HANDLER *writeMethod_ = nullptr; ///< exports Squid bytes
+
+    uint64_t bytes_read = 0; ///< aggregate bytes read
+    uint64_t bytes_written = 0; ///< aggregate bytes written
```

Comment:
Since we are (correctly!) changing the type of these data members, and have to do it in a rather subtle way that may hide certain bugs from the compiler, please _rename_ them. This renaming will also allow us to follow the current Squid naming style better and be more consistent with the two other closely related data member names in this class.

```suggestion
    uint64_t totalBytesRead_ = 0; ///< aggregate bytes received
    uint64_t totalBytesWritten_ = 0; ///< aggregate bytes sent
```

I like using synonyms instead of saying "foo is foo", but I do not insist on the above comment adjustment. Here is a version without it:

```suggestion
    uint64_t totalBytesRead_ = 0; ///< aggregate bytes read
    uint64_t totalBytesWritten_ = 0; ///< aggregate bytes written
```

Needless to say, the other PR code will need to be adjusted accordingly, but it is a straightforward (and useful) change.